docs/schemas/domain/IfcRailDomain/Types/IfcRailwayPartTypeEnum.md

```diff
@@ -2,56 +2,83 @@
 
 The _IfcRailwayPartTypeEnum_ defines the range of different types of railway part that can be specified.
 
-The spatial structure of a railway is sub-divided into several elements for lateral, vertical and longitudinal decomposition:
+A railway line can be spatially organised into several parts, using a vertical, longitudinal and later criteria for its division. There is not one standardised way of spatially divide the line, as this depends on the use case. The _IfcRailwayPart_'s that can be used to spatially organise a line are:
```

Comment:
We seem to be mixing the word 'line' and 'track'. Please double check that the wording is used correctly and consistently. I can't really judge as a non-domain expert.

Reply:
might need another look from @czapplitec on this. In general, the track is a part of the line.

Reply:
Hi Thomas!

Obviously the English terms "line" and "track" are considered to be selfevident by the English speaking railway community. The English wikipedia is very weak on rail terminology. Both the German and French wikipedia contain specific pages for "Eisenbahnstrecke" or "Ligne de chemin de fer". Not so in English.

This is an automatic translation of parts of the German "Eisenbahnstrecke" wikipage:

"Railway line is called the connection of places with railway tracks."

"The route of the railway line may be equipped with one or more tracks, signaling equipment, elevated structures, mileage, switches and crossings, as well as stations and stops (stops) for passenger and freight traffic, an overhead line for operation with electric locomotives and electric railcars."

So it can be safely assumed that there is some kind of a hierachical relationship between "railway line" and "railway track" described by sentences like the following:

A railway line consists of one or more tracks.
A railway track belongs to one railway line. 
Two running trains can be at the same point of a railway line at the same time (if this line has at least two parallel tracks). 
On one track two trains can never be at the same point at the same time. Otherwise it would be an accident.

I think you get the idea ;-) Please note that things become more complicated if one looks into junctions between two or more lines. Then it can happen that one track belongs to more than one line. It helps a lot if one thinks about railway as a network or even better as set of interconnected networks on different aggregation levels. UIC "macro level" designates a network of railway lines, UIC "micro level" designates a network of railway tracks.

Anyhow, the sentence with the train accident is always true, also in the complicated cases.

I have an UK rail engineering dictionary somewhere in the basement. I will try to add an original English definition for railway line and railway track once I find it.

Reply:
The relevant takeaway for the current discussion that the "line" and "track" terms indicate somehow level of detail aspects.

E.g. in the very early stages of railway design the engineer could work with the "railway line" alignment only. In later more detailled stages the engineers will work with more detailed "railway  track" alignments.

So your observation of "mixing 'line' and 'track'" could simply mean that for one phase people have to organize the space along the "railway line" and in later phases they have to organize the space along "railway tracks".